apps/io-session-manager/api/external.yaml

```diff
@@ -0,0 +1,182 @@
+openapi: 3.0.1
+info:
+  title: Token instrospection API
+  description: Collection of endpoints to interact with user's auth session.
+  version: 16.7.0
+servers:
+  - url: https://api-app.io.pagopa.it/api/v1
+security:
+  - Bearer: []
+paths:
+  /user-identity:
+    get:
+      summary: Session token introspection
+      description: Introspect the session token and returns the related user identity
+      operationId: getUserIdentity
+      responses:
+        "200":
+          description: Found.
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/UserIdentityWithTokens"
+              example:
+                family_name: Rossi
+                fiscal_code: TMMEXQ60A10Y526X
+                name: Mario
+                spid_email: preferred@example.com
+                spid_level: https://www.spid.gov.it/SpidL2
+                date_of_birth: "2007-07-21"
+                session_token: qwertyuiopasdfghjklzxcvbnm1234567890
+                wallet_token: qwertyuiopasdfghjklzxcvbnm1234567890
+                myportal_token: qwertyuiopasdfghjklzxcvbnm1234567890
+                bpd_token: qwertyuiopasdfghjklzxcvbnm1234567890
+                zendesk_token: qwertyuiopasdfghjklzxcvbnm1234567890
+                fims_token: qwertyuiopasdfghjklzxcvbnm1234567890
```

Comment:
Do we need to expose these tokens?

Reply:
Moreover, if the Zendesk token is required, we should invoke the getZendeskToken method to generate it.

Reply:
removed in [2787e7d](https://github.com/pagopa/io-auth-n-identity-domain/pull/251/commits/2787e7d0b7f4ffee980cb029e9b82c337cd6050a)